.github/workflows/build_docs.yml

```
@@ -67,12 +75,12 @@ jobs:
         echo "::endgroup::"
         echo "::group::Install TorchAudio"
         conda install --quiet --yes cmake>=3.18.0 ninja
-        pip3 install --progress-bar off -v -e . --no-use-pep517
+        pip install --progress-bar off -v . --no-build-isolation
 
         # TODO: Need to rely on torchcodec instead of building ffmpeg from source.
         echo "::endgroup::"
         echo "::group::Build FFmpeg"
-        .github/scripts/ffmpeg/build_gpu.sh
+        conda install -q -y ffmpeg
```

Notice:
```
  RuntimeError: Could not load libtorchcodec. Likely causes:
            1. FFmpeg is not properly installed in your environment. We support
               versions 4, 5, 6 and 7.
```
but conda install version 8. So:
```suggestion
        conda install -q -y ffmpeg<=7
```